components/FiltersSection/__tests__/Categories.test.tsx

```diff
@@ -18,6 +18,13 @@ const mockCategories = [
   },
 ];
 
+jest
+  .spyOn(GlobalContext, 'useGlobalStore')
+  .mockReturnValue({
+    state: { brands: [], categories: mockCategories, cart: [] },
+    dispatch: jest.fn,
+  });
+
 const MockCategory = () => {
   const [mockValue, setMockValue] = useState<{
     [key: string]: {
```

Comment:
Please avoid this approach of mocking the component with another component, it beats the purpose of testing